README.md

```diff
@@ -1,12 +1,14 @@
 [![Build Status](https://travis-ci.org/OCA/purchase-workflow.svg?branch=7.0)](https://travis-ci.org/OCA/purchase-workflow)
```

Comment:
I'm wondering if we should rename this to "Travis Status" to differentiate it from the runbot one...